lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -13,1517 +13,19 @@
  */
 package io.trino.plugin.base.security;
 
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Sets;
-import io.trino.spi.QueryId;
-import io.trino.spi.connector.CatalogSchemaName;
-import io.trino.spi.connector.CatalogSchemaRoutineName;
-import io.trino.spi.connector.CatalogSchemaTableName;
-import io.trino.spi.connector.SchemaTableName;
-import io.trino.spi.security.AccessDeniedException;
-import io.trino.spi.security.Identity;
-import io.trino.spi.security.PrincipalType;
-import io.trino.spi.security.Privilege;
 import io.trino.spi.security.SystemAccessControl;
-import io.trino.spi.security.SystemSecurityContext;
-import io.trino.spi.security.TrinoPrincipal;
-import io.trino.spi.security.ViewExpression;
-import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-
-import javax.security.auth.kerberos.KerberosPrincipal;
 
 import java.io.File;
-import java.util.Collection;
-import java.util.EnumSet;
-import java.util.List;
 import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.stream.Stream;
-
-import static com.google.common.io.Files.copy;
-import static io.trino.plugin.base.security.FileBasedAccessControlConfig.SECURITY_CONFIG_FILE;
-import static io.trino.plugin.base.security.FileBasedAccessControlConfig.SECURITY_REFRESH_PERIOD;
-import static io.trino.spi.function.FunctionKind.AGGREGATE;
-import static io.trino.spi.function.FunctionKind.SCALAR;
-import static io.trino.spi.function.FunctionKind.TABLE;
-import static io.trino.spi.function.FunctionKind.WINDOW;
-import static io.trino.spi.security.PrincipalType.USER;
-import static io.trino.spi.security.Privilege.UPDATE;
-import static io.trino.spi.testing.InterfaceTestUtils.assertAllMethodsOverridden;
-import static io.trino.spi.type.VarcharType.VARCHAR;
-import static java.lang.String.format;
-import static java.lang.Thread.sleep;
-import static java.util.Objects.requireNonNull;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.assertj.core.util.Files.newTemporaryFile;
-import static org.testng.Assert.assertEquals;
 
 public class TestFileBasedSystemAccessControl
+        extends BaseFileBasedSystemAccessControlTest
 {
-    private static final Identity alice = Identity.forUser("alice").withGroups(ImmutableSet.of("staff")).build();
```

Comment:
Can we have this extraction of test as a preparatory commit - so that we don't lose the changes history.